src/shared/sandboxRequest.ts

```diff
@@ -143,11 +149,68 @@ export async function getSrcIdByName(conn: Connection, sandboxName: string): Pro
   }
 }
 
+export async function querySandboxProcessFeatures(
+  conn: Connection,
+  identifier: string,
+  isId: boolean
+): Promise<string | undefined> {
+  const query = isId
+    ? `SELECT Features FROM SandboxProcess WHERE Id = '${identifier}'`
+    : `SELECT Features FROM SandboxProcess WHERE SandboxName = '${identifier}'`;
+
+  try {
+    const result = await conn.singleRecordQuery<{ Features: string }>(query, { tooling: true });
+    return result.Features;
+  } catch (err) {
+    throw cloneMessages.createError('error.featuresQueryFailed', [identifier], [], err as Error);
+  }
+}
+
+export async function querySandboxInfoFeatures(
+  conn: Connection,
+  identifier: string,
+  isId: boolean
+): Promise<string | undefined> {
+  const query = isId
+    ? `SELECT Features FROM SandboxInfo WHERE Id = '${identifier}'`
+    : `SELECT Features FROM SandboxInfo WHERE SandboxName = '${identifier}'`;
+
+  try {
+    const result = await conn.singleRecordQuery<{ Features: string }>(query, { tooling: true });
+    return result.Features;
+  } catch (err) {
+    throw cloneMessages.createError('error.featuresQueryFailed', [identifier], [], err as Error);
+  }
+}
```

Comment:
If there's no query for SandboxInfo in the `Org` class I would add this code there.